tests/unit/compose/docker-compose-legacy-auth_test.go

```diff
@@ -296,6 +297,89 @@ func (s *commonServicesLegacyAuthDockerComposeTest) TestServiceEnvironment() {
 				"TEAMS_API_MONGODB_URI=mongodb://root:test-secret@mongodb.local/?authSource=admin",
 			},
 		},
+		{
+			"ffDisableInvitationsFiftyoneApp",
```

Comment:
Add new test cases to validate that when `FEATURE_FLAG_ENABLE_INVITATIONS` overridden to `false`, that it is false as expected.